underscore.util.strings.js

```diff
@@ -103,6 +103,14 @@
       return a.join('');
     },
 
+    // Checks whether a string is "trueish"
+    isTrueish: function(v) {
+      if(_.isBoolean(v)) {
+          return v;
+      }
+      return !!JSON.parse((v || "false").toLowerCase());
+    },
```

Comment:
That would certainly be more flexible! My use case was specifically for optional strings but this change would be nice since it would still cover that.
We could replace `_.isString` with `typeof v == 'string'` as well to reduce function calls

Reply:
Replacing `_.isString(v)` by `typeof v === 'string'` would save a function call, but it would only work for primitive strings. `_.isTrueish(Object('false'))` would return `true` in that case.

Reply:
I'm not sure what to do about the function call, but I do like this solution so I copied it into the changes.

Reply:
Good point. To make non-strings entirely consistent with strings, you could convert all values to string first. That can be done with `'' + v`. I don't think you'll get the same notation for objects as with JSON in that way, but I do think they'll evaluate to `false`.

What do you think?

Reply:
That seems to work! It also removes the `isString` check since they're now all strings:
```javascript
isTrueish: function(v) {
  v = `${v}`;
  return !!+v || v.toLowerCase() === 'true';
}
```

Are template strings allowed into the repo? They're [supported by all the important browsers](https://caniuse.com/template-literals)

Reply:
Contrib (and Underscore proper) currently only uses ES3 syntax, so anything newer than that would be a breaking change, so please don't do that. In this case, it is not even shorter, though.

Reply:
Yep, being short wasn't the concern! It's IMO a little clearer and less hacky looking than `'' + v`, but I figured compatibility would be important 👍️